test/e2e/clustertask/start_test.go

```diff
@@ -232,4 +232,31 @@ Waiting for logs to be available...
 			t.Fatalf("-got, +want: %v", d)
 		}
 	})
+
+	t.Run("Start TaskRun using tkn ct start with --use-taskrun option", func(t *testing.T) {
+		// Get last TaskRun for read-clustertask
+		lastTaskRun := builder.GetTaskRunListWithName(c, "read-clustertask", true).Items[0]
+
+		// Start TaskRun using --use-taskrun
+		tkn.MustSucceed(t, "ct", "start", "read-clustertask",
+			"--use-taskrun",
+			""+lastTaskRun.Name,
```

Comment:
What is the reason for concatenation here? Why not just use `lastTaskRun.Name` instead of `""+lastTaskRun.Name`?

Reply:
it was a nit :sweat_smile: